app/view_models/nsm/v1/work_item_summary.rb

```diff
@@ -25,22 +26,23 @@ def header
       end
 
       def table_fields
-        work_item_data.map do |data|
+        WORK_TYPES.map do |work_type|
+          data = data_for(work_type)
           format_row(data)
         end
       end
 
       def footer
         format_row(
-          [t('total', numeric: false)] + summed_values(work_items, periods: false),
+          [t('total', numeric: false)] + summed_values(work_items, work_items, periods: false),
```

Comment:
Why is `work_items` being passed twice?

Reply:
It was because I changed the signature of `summed_values` to require both e "claimed work items" and a "allowed work items" - for the purposes of the footer, both are just "all work items". But it was clunky, so I've updated the signature to make the second one optional and default to the first one.